articles/azure-monitor/logs/basic-logs-configure.md

```diff
@@ -17,7 +17,7 @@ Setting a table's [log data plan](log-analytics-workspace-overview.md#log-data-p
 By default, all tables in your Log Analytics are Analytics tables, and available for query and alerts. 
 You can currently configure the following tables for Basic Logs:
 
-- All tables created with the [Data Collection Rule (DCR)-based logs ingestion API.](logs-ingestion-api-overview.md) 
+- All tables custom log tables created with the [Data Collection Rule (DCR)-based logs ingestion API.](logs-ingestion-api-overview.md) 
```

Comment:
It should be:
All custom tables created with or converted to the [Data Collection Rule (DCR)-based logs ingestion API.](logs-ingestion-api-overview.md)